raft/tracker/tracker.go

```diff
@@ -166,10 +166,34 @@ func (p *ProgressTracker) Committed() uint64 {
 	return uint64(p.Voters.CommittedIndex(matchAckIndexer(p.Progress)))
 }
 
-// Visit invokes the supplied closure for all tracked progresses.
+func insertionSort(sl []uint64) {
```

Comment:
move this to a common util pkg? no need for two insertionSort.

Reply:
+1

Reply:
I intentionally didn't do that (https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s). Just to double check, are you suggesting I make an `insertionsortutil` package that only includes this method? Because it won't be able to include much more than because quorum's dependency tree is quite small and that's a good thing (plus, lots of our existing util-like code actually depends on raft).